### ISSUE TITLE
link in email doesn't link to ticket.

### ISSUE THREAD

**Issue by pieter-groeneweg** [opensupports/opensupports#672]
when receiving an email on ticket creation or responses, the button has a link like:
`https://opensupport.domain.com/check-ticket/123456/email@emaildomain.com`

that seems generated in the email template by:
`{{url}}/check-ticket/{{ticketNumber}}/{{to}}`

Expected behaviour.
When clicking that link, one would expect to go to the ticket 123456. 

Actual behaviour.
When already logged in it takes user to the dashboard.
When not logged in, user is presented with login page and after taken to dashboard.

The API uses a csrf_token or csrf_userid.. shouldn't it have something like that?

**Comment by mredigonda** [comment]
The `/check-ticket` path is meant to be used only when there is no user system enabled, do you have it enabled or disabled?

**Comment by pieter-groeneweg** [comment]
Thank you for your answer. 
I see.... I have user system enabled. But the link is still there in the emails... I cannot find a condition in there. But it would be nice if one ended up at the ticket "promised" in the email link.
So even after having to log in, the user will be directed to his/her issue.
As now a user expects to see the ticket on clicking the link.

Tx,

**Comment by mredigonda** [comment]
Yes, it is true, that would be great, but the fact that the link to `/check-ticket` is present in the mails even when the user system is enabled means that you got to a state we were not really aware of.
It would be great to have logs to investigate the issue that would let us know what steps you took, but so far we don't have a system like that.
Thanks for reporting this!

**Comment by pieter-groeneweg** [comment]
I need to find some other things out now.. I switched with user system enabled to disabled... this triggered other errors (https://github.com/opensupports/opensupports/issues/498)
Therefore I reverted back... now reporters do not receive any mails anymore at all....

No error messages :(
